dataset-snippets/snippets_setup.R

```diff
@@ -0,0 +1,20 @@
+lapply(c('viridis', 'ggthemes', 'skimr'),
```

Comment:
Does anything need to be shared between the setup scripts, or should they just be the same thing for ease of maintenance/use?

Reply:
At this point I think its best to keep the setup for the Dataset Builder snippets and the SQL snippets separate.

A future PR will add more default parameters to the SQL setup snippet, etc.